src/gui/guiFormSpecMenu.cpp

```diff
@@ -103,6 +105,7 @@ GUIFormSpecMenu::GUIFormSpecMenu(JoystickController *joystick,
 	m_tsrc(tsrc),
 	m_client(client),
 	m_formspec_prepend(formspecPrepend),
+	m_texture_pool(new TexturePool()),
```

Comment:
You have to delete this in the destructor (line 148).

Reply:
Whoops, you're right. I need to be more careful, been making some really dumb errors on this PR...